Tasks/AppStoreRelease/task.json

```diff
@@ -135,13 +135,36 @@
             "helpMarkDown": "Used with two-step verification.  Create by running: 'fastlane spaceauth -u [email]'",
             "visibleRule": "isTwoFactorAuth = true"
         },
+        {
+            "name": "distributeOnly",
```

Comment:
Could you please update docs for all new/changes inputs?

Reply:
Done